src/components/FindMovie/FindMovie.tsx

```diff
@@ -1,7 +1,77 @@
-import React from 'react';
+import classNames from 'classnames';
+import React, { ChangeEvent, FormEvent, useState } from 'react';
+import { getMovie } from '../../api';
+import { Movie } from '../../types/Movie';
+import { MovieData } from '../../types/MovieData';
+import { MovieCard } from '../MovieCard';
 import './FindMovie.scss';
 
-export const FindMovie: React.FC = () => {
+export function normalizeMovie(data: MovieData): Movie {
+  const {
+    Title,
+    Plot,
+    Poster,
+    imdbID,
+  } = data;
+
+  // eslint-disable-next-line max-len
+  const defaultPoster = 'https://via.placeholder.com/360x270.png?text=no%20preview';
+  const imdbUrl = `https://www.imdb.com/title/${imdbID}`;
+
+  return ({
+    title: Title,
+    description: Plot,
+    imgUrl: Poster === 'N/A' ? defaultPoster : Poster,
+    imdbUrl,
+    imdbId: imdbID,
+  });
+}
+
+type Props = {
+  query: string,
+  onSearch: (value: string) => void;
+  onAddMovie: (newMovie: Movie) => void
+};
+
+export const FindMovie: React.FC<Props> = ({
+  query,
+  onSearch,
+  onAddMovie,
+}) => {
+  const [movie, setMovie] = useState<Movie | null>(null);
+  const [loading, isLoading] = useState(false);
```

Comment:
it would be better use the name conventions
```suggestion
  const [loading, setLoading] = useState(false);
```